hyperspy/logger.py

```diff
@@ -62,6 +66,44 @@ def set_log_level(level):
     DEBUG:hyperspy.gui:Current ETS toolkit set to: null
 
     """
-    import logging
-    logging.basicConfig()  # Does nothing if already configured
-    logging.getLogger('hyperspy').setLevel(level)
+    logger = initialize_logger('hyperspy')
+    logger.setLevel(level)
+
+class ColoredFormatter(logging.Formatter):
+    """Colored log formatter."""
+    grey = "\x1b[38;20m"
```

Comment:
For future reference, where does these color code come from?

Reply:
The color codes are taken from here. 
https://stackoverflow.com/questions/384076/how-can-i-color-python-logging-output